skimage/io/tests/test_freeimage.py

```diff
@@ -112,5 +113,13 @@ def test_metadata():
     assert meta[1][('EXIF_MAIN', 'Software')].startswith('I')
 
 
+@skipif(not FI_available)
+def test_collection():
+    pattern = [os.path.join(data_dir, pic)
+               for pic in ['camera.png', 'color.png']]
```

Comment:
I think we want to test here as well that a multi-page tiff is read correctly. I'm concerned that some plugins will read the full 3D image in a multipage tiff if there is no img_num specified... What do you think?